### ISSUE TITLE
dSYM generation should be off for all debug builds

### ISSUE THREAD

**Issue by michaeleisel** [fastred/Optimizing-Swift-Build-Times#21]
The use case provided for why one would have dSYM generation on is rare and should be opt-in. A lot of people leave dSYM generation on and don't realize it's adding 5+ seconds to every incremental build.

**Comment by fastred** [comment]
Agree. Would you like to create a PR with an improved explanation in that section?

**Comment by michaeleisel** [comment]
Eh, I'm too lazy
